demo.Dockerfile

```diff
@@ -1,7 +1,15 @@
+FROM jaegertracing/all-in-one:latest AS jaeger-builder
+
 FROM quay.io/konveyor/analyzer-lsp
 
+COPY --from=jaeger-builder /go/bin/all-in-one-linux /usr/bin/
+
 WORKDIR /analyzer-lsp
 
 COPY rule-example.yaml /analyzer-lsp/rule-example.yaml
 COPY examples /analyzer-lsp/examples
 COPY open-source-libs.txt /analyzer-lsp/open-source-libs.txt
+
+EXPOSE 5775/udp 6831/udp 6832/udp 5778 16686 14268 9411
+
+ENTRYPOINT ["sh", "-c", "all-in-one-linux & sleep 5 && konveyor-analyzer --enable-jaeger=true --jaeger-endpoint=http://localhost:14268/api/traces && curl -o traces.json http://localhost:16686/api/traces?service=analyzer-lsp"]
```

Comment:
this `--jaeger-endpoint` is already the default, no need to explicitely pass https://github.com/konveyor/analyzer-lsp/blob/7cb1d667a4088a4ba11711491ff98987ce80c3ab/cmd/analyzer/main.go#L58

Reply:
--enable-jaeger=true already there.. I think I just need to remove the --jaeger-endpoint flag?

Reply:
@Parthiba-Hazra I meant in the analysis CLI here https://github.com/konveyor/analyzer-lsp/blob/7cb1d667a4088a4ba11711491ff98987ce80c3ab/cmd/analyzer/main.go#L57

Reply:
Done.. thanks for the guidance :)